spec/command/users_spec.rb

```diff
@@ -52,5 +52,19 @@
         invoke
       end
     end
+    describe_command 'user rotate_api_key --user non-existing' do
+      before do
+      expect(RestClient::Request).to receive(:execute).with({
+            method: :head,
+            url: "https://core.example.com/api/resources/#{account}/user/non-existing",
+            headers: {authorization: "fakeauth"},
+            username: "dknuth",
```

Comment:
what is `dknuth`? should it stand for something?

Reply:
`dknuth` is the user that is hard-coded in the mock service of Spec tests in this repo.
I now see that it is assigned to a global variable names `username` , so I can change it to make it more readable

Reply:
thanks, that will be good.